browser/profiles/brave_profile_manager_browsertest.cc

```diff
@@ -30,6 +28,14 @@
 #include "brave/components/ipfs/features.h"
 #endif
 
+#if defined(OS_ANDROID)
+#include "chrome/test/base/android/android_browser_test.h"
+#else
+#include "chrome/test/base/in_process_browser_test.h"
```

Comment:
this is also not correct, see above

Reply:
I don't get it, the include is in `#else` and the first condition is `#if defined(OS_ANDROID)` so here is will not be included on Android. Is that right?

Reply:
yep, I missed the else in both of these